lightning/src/routing/scoring.rs

```diff
@@ -15,22 +15,22 @@
 //! # Example
 //!
 //! ```
-//! # extern crate bitcoin;
-//! #
-//! # use lightning::routing::gossip::NetworkGraph;
-//! # use lightning::routing::router::{RouteParameters, find_route};
-//! # use lightning::routing::scoring::{ProbabilisticScorer, ProbabilisticScoringFeeParameters, ProbabilisticScoringDecayParameters};
-//! # use lightning::sign::KeysManager;
-//! # use lightning::util::logger::{Logger, Record};
-//! # use bitcoin::secp256k1::PublicKey;
-//! #
-//! # struct FakeLogger {};
-//! # impl Logger for FakeLogger {
-//! #     fn log(&self, record: &Record) { unimplemented!() }
-//! # }
-//! # fn find_scored_route(payer: PublicKey, route_params: RouteParameters, network_graph: NetworkGraph<&FakeLogger>) {
-//! # let logger = FakeLogger {};
-//! #
+//!  extern crate bitcoin;
```

Comment:
I had to enable this test as it was failing for me locally when running ci checks.. should I move this to a different commit?

Reply:
The `#` marks simply hide the code from the rust docs. It shouldn't affect the test result, so it should remain as it was. Do you have the failure message?